site/routes/definitions.js

```diff
@@ -56,6 +69,16 @@ export const routeDefinitions : Array<RouteDefinition> = [
     gen: state => state.events.map(({ startDateTime: { date, month, year }, slug }) => ({ date, month, year, slug })),
   },
   {
+    title: ({ category }) => 'Meet our team' + (category !== 'everyone' ? ` (${category})` : ''),
+    key: 'badgers',
+    route: 'about-us/people/{category?}/{page?}',
+    defaults: { category: 'everyone', page: 1 },
+    urlEncode: (_, key, val) => (val === 'ux & design' ? 'ux-design' : encodeURIComponent(val)),
+    urlDecode: (_, key, val) => (val === 'ux-design' ? 'ux & design' : decodeURIComponent(val)),
```

Comment:
Could we handle this in the graphql layer? I.e. produce a slug for categories there?